libafl/src/observers/map.rs

```diff
@@ -491,7 +491,7 @@ where
     }
 }
 
-/// A Simple iterator calling `MapObserver::get_mut`
+/// A Simple iterator calling `MapObserver::get`
 #[derive(Debug)]
 pub struct MapObserverSimpleIteratorMut<'a, O>
```

Comment:
It might be better to just delete this type altogether, to be honest. I think this is a vestige of early LibAFL code that is generally unused, and has seriously poor performance in a lot of applications (including yours).